src/Tribe/Views/V2/Messages.php

```diff
@@ -94,6 +94,11 @@ public static function for_key( string $key, ...$values ) {
 				'There were no results found for <strong>"%1$s"</strong> this month. Try searching next month.',
 				'the-events-calendar'
 			),
+			// translators: the placeholder is the formatted date string, e.g. "February 22, 2020".
+			'day_no_results_found' => __(
```

Comment:
:warning: Array double arrow not aligned correctly; expected 13 space(s) between "'day_no_results_found'" and double arrow, but found 1.

<sub>hash: 0144f01f399f9d916147ec63524952e3</sub>